lib/core/src/lib/viewer/components/viewer.component.html

```diff
@@ -44,8 +44,8 @@
                         <mat-icon>navigate_before</mat-icon>
                     </button>
                     <img class="adf-viewer__mimeicon"
-                         [alt]="mimeType"
-                         [src]="mimeType | adfMimeTypeIcon"
+                         [alt]="(originalMimeType ? originalMimeType : mimeType)"
+                         [src]="(originalMimeType ? originalMimeType : mimeType) | adfMimeTypeIcon"
```

Comment:
can be just 
(originalMimeType || mimeType) | adfMimeTypeIcon

Reply:
Updated!